mlrun/projects/project.py

```diff
@@ -576,6 +577,9 @@ def __init__(
         self.disable_auto_mount = disable_auto_mount
         self.default_image = default_image
 
+        self._build = None
+        self.build = build
```

Comment:
why not lazy load the build upon get?
having both properties set here that way might be confusing for their purposes

Reply:
Can indeed let go of `_build`, but must keep the `build` parameter because of `to_dict`/`from_dict` reasons - it analyzes the parameters of the `__init__` method to figure out what to (de)serialize.

Reply:
TIL